test/e2e/main/main_test.go

```diff
@@ -817,3 +818,61 @@ func throwHTTPError(resp *http.Response) {
 	errorBytes, _ := ioutil.ReadAll(resp.Body)
 	panic(fmt.Errorf("unexpected HTTP status code (%d): %s", resp.StatusCode, string(errorBytes)))
 }
+
+func TestCacheCR(t *testing.T) {
+	spec := tutils.DefaultSpec(testKube)
+	name := strcase.ToKebab(t.Name())
+	spec.Name = name
+	testKube.CreateInfinispan(spec, tutils.Namespace)
+	testKube.WaitForInfinispanPods(1, tutils.SinglePodTimeout, spec.Name, tutils.Namespace)
+	testKube.WaitForInfinispanCondition(spec.Name, spec.Namespace, ispnv1.ConditionWellFormed)
+
+	cacheNameForTemplateName := "testtemplatename"
+	cacheNameForTemplateXML := "testtemplatenamexml"
```

Comment:
I would remove these two vars and just specify the string inline for each call to `createCacheCR`

Reply:
I would recommend to give more meaningful names, i.e. `cache-with-xml-template` and `cache-with-static-template`